src/Neo/SmartContract/JsonSerializer.cs

```diff
@@ -50,10 +50,7 @@ public static JToken Serialize(StackItem item)
                     }
                 case Integer num:
                     {
-                        var integer = num.GetInteger();
-                        if (integer > JNumber.MAX_SAFE_INTEGER || integer < JNumber.MIN_SAFE_INTEGER)
```

Comment:
Hello, #2498.

Reply:
He put it in the JNumber. Different exception, but will fail as well.

Reply:
It was moved to the constructor, that was the bug, when we use parse we allowed it...